src/index.tsx

```diff
@@ -1,14 +1,51 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  HashRouter, Navigate, Route, Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { PeopleTable } from './components/PepopleTable/PeopleTable';
+import { NotFoundPage } from './components/NotFoundPage/NotFoundPage';
 import { App } from './App';
 
 createRoot(document.getElementById('root') as HTMLDivElement)
   .render(
-    <Router>
-      <App />
-    </Router>,
+    <HashRouter>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+
+          <Route path="/home" element={<Navigate to="/" replace />} />
+          <Route path="people">
+            <Route
+              index
+              element={(
+                <>
+                  <PeoplePage />
+                  <PeopleTable />
```

Comment:
I am not sure if it has to be two separate components. I guess PeopleTable must be inside of PeoplePage. But maybe its not necessary.

Reply:
Yes, PeopleTable should be inside `PeoplePage`